workspaces/src/types/mod.rs

```diff
@@ -10,6 +10,9 @@ pub(crate) mod gas_meter;
 #[cfg(feature = "interop_sdk")]
 mod sdk;
 
+mod token;
+pub use token::NearToken;
+
```

Comment:
1. I would move it below to replace `Balance` type alias.
2. I don't think there is a value in having `token.rs` file. Let's drop it.